src/js/components/inspectors/Property.jsx

```diff
@@ -106,7 +103,6 @@ var Property = React.createClass({
             <div>
               <input type="color" value={!disabled && value} disabled={disabled}
                 onChange={this.handleChange} />
-              <ContentEditable {...props} />
```

Comment:
Just got caught these. Is there a reason they were removed? Users quite liked being able to manually enter hexcodes/numbers rather than being forced to use a color picker/slider.